code-challenge-android-kotlin/app/build.gradle

```diff
@@ -36,6 +36,12 @@ dependencies {
     implementation "io.reactivex.rxjava2:rxandroid:2.0.2"
     implementation 'com.github.bumptech.glide:glide:4.6.1'
 
+    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:$coroutines_version"
```

Comment:
Why did you choose to use Kotlin Coroutines?

Reply:
In my opinion, they are simpler to use than rxjava.